README.md

```diff
@@ -123,7 +123,7 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-48-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
```

Comment:
I'm not sure if this is impacted by what was deleted in the `.all-contributors` file, but to ensure this works, the only update you should see is an increase in the contributor number.